.clang-format

```diff
@@ -22,31 +22,31 @@ BreakBeforeBinaryOperators: false
 BreakBeforeBraces: WebKit
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: true
-ColumnLimit:     120
-CommentPragmas:  '^ IWYU pragma:'
+ColumnLimit: 120
+CommentPragmas: "^ IWYU pragma:"
 ConstructorInitializerAllOnOneLineOrOnePerLine: true
 ConstructorInitializerIndentWidth: 4
 ContinuationIndentWidth: 4
 Cpp11BracedListStyle: true
 DerivePointerAlignment: false
-DisableFormat:   false
+DisableFormat: false
 ExperimentalAutoDetectBinPacking: false
 FixNamespaceComments: true
-ForEachMacros:   [ Q_FOREACH,  BOOST_FOREACH ]
+ForEachMacros: [Q_FOREACH, BOOST_FOREACH]
 IncludeBlocks: Regroup
 IncludeCategories:
-  - Regex:           '^".*"$'
-    Priority:        1
```

Comment:
Even though in some cases it makes sense to have space-alignment, I doubt it's common, and the benefit of having uniform codestyle in my opinion is more than damage in places like this